src/content/docs/apm/agents/nodejs-agent/installation-configuration/es-modules.mdx

```diff
@@ -78,6 +78,13 @@ You can add the loader to your application by using the Node.js `--experimental-
 node --experimental-loader newrelic/esm-loader.mjs your-program.js
 ```
 
+## Custom instrumentation [#custom-instrumentation]
+The Node.js APM agent supports adding custom instrumentation to your ES module applications.
+You can create instrumentation in ES module applications using most of the [instrumentation registration methods on the API](/docs/apm/agents/nodejs-agent/api-guides/nodejs-agent-api/#custom-instrumentation-api).
+Due to the nature of ES module imports, we are unable to support `newrelic.instrumentLoadedModule`. ES module bindings are not assignable, which prevent the agent from replacing a module's exported members with instrumentation after the module has been loaded.
+
+To see a demonstration of how to use the custom instrumentation API in an ES module application, check out our [example app](https://github.com/newrelic/newrelic-node-examples/tree/main/esm-app) in GitHub.
```

Comment:
do we also want to put the link to this app at the bottom of the page with the other links?